casepro/pods/registry.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+import importlib
+
+
+def get_class_from_string(string):
+    '''
+    Given a class specified by a string, this will return that class.
+    '''
+    module_name, cls_name = string.rsplit('.', 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, cls_name)
+
+
+def load_pod(index, config):
+    '''
+    Given the index of the pod, and the config dictionary for the pod, this
+    returns the instance of that pod.
+    '''
+    class_name = config.pop('type')
+    config['index'] = index
+    return get_class_from_string(class_name)(config)
+
+
+pods = [
```

Comment:
I'm wondering whether this should be a function rather than a global variable in case we want to make it more dynamic later. Not sure how to make that work with URL patterns though.